sysutils/peco/Portfile

```diff
@@ -15,53 +14,18 @@ long_description \
 license         MIT
 
 maintainers     {kimuraw @kimuraw} openmaintainer
-platforms       darwin
```

Comment:
OK, sorry, I noticed in the documentation that you define this variable there. But apart from the general "problem" of this keyword that's valid elsewhere, I see no particular reason that makes the go packages special in this respect.

Reply:
> But apart from the general "problem" of this keyword that's valid elsewhere, I see no particular reason that makes the go packages special in this respect.

Sorry, I'm not sure I understand your objection.

Also, depending on when and where you looked at the docs, the info might have been slightly out of date: Yesterday I added support for `GOOS` and `GOARCH` to the portgroup. In principle all Go packages can be built for darwin, linux, and freebsd (and others we don't care about) (=`GOOS` / `platforms`) with archs `386` or `x86_64` (=`GOARCH` / `archs`). I thought it would make sense to set all of these in the portgroup and let individual ports pare down the options in case some things are not supported.